chef/automate
# Cli milestone part-1

### :nut_and_bolt: Description: What code changed, and why?



### :chains: Related Resources

### :+1: Definition of Done

### :athletic_shoe: How to Build and Test the Change

1. check out the repo
2. uncomment the lines from 111 to 151 in components/automate-cli/cmd/chef-automate/upgrade.go
3. build automate-cli, automate-deployment, automate-ui 
4. start all the services(start_all_services)
5. once automate is up and running, run `chef-automate upgrade status`
6. run `chef-automate upgrade run --major`


### :white_check_mark: Checklist

**All PRs** must tick these:

With occasional exceptions, all PRs **from Progress employees** must tick these:


All PRs **from Progress employees** should tick these if appropriate:


*Please add a note next to any checkbox above if you are NOT ticking it.*

### :camera: Screenshots, if applicable
